xml/System.Security.Cryptography/ECDiffieHellman.xml

```diff
@@ -1051,10 +1051,20 @@ The algorithm-specific key import failed.</exception>
         <Parameter Name="passwordBytes" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="input">To be added.</param>
-        <param name="passwordBytes">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="input">The PEM text of the encrypted key to import.</param>
+        <param name="passwordBytes">The bytes to use as a password when decrypting the key material.</param>
+        <summary>Imports an encrypted RFC 7468 PEM-encoded private key, replacing the keys for this object.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ The password bytes are passed directly into the Key Derivation Function (KDF) used by the algorithm indicated by <c>pbeParameters</c>. This enables compatibility with other systems that use a text encoding other than UTF-8 when processing passwords with PBKDF2 (Password-Based Key Derivation Function 2).
+  Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels are found, an exception is thrown to prevent importing a key when the key is ambiguous.
+ This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.
```

Comment:
Need to make sure the DocsPortingTool detects `<c></c>` tags in remarks and substitutes them with backticks:
```suggestion
 The password bytes are passed directly into the Key Derivation Function (KDF) used by the algorithm indicated by `pbeParameters`. This enables compatibility with other systems that use a text encoding other than UTF-8 when processing passwords with PBKDF2 (Password-Based Key Derivation Function 2).
  Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels are found, an exception is thrown to prevent importing a key when the key is ambiguous.
 This method supports the `ENCRYPTED PRIVATE KEY` PEM label.
```